index.html

```diff
@@ -23,53 +23,347 @@
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="me">
+        <div class="part1ml">
+          <h2 class="abm">
+            About <br />
+            Myself
+          </h2>
+
+          <p class="text">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="connect"><a href="#">LET'S CONNECT</a></p>
+          <div class="social-icons">
+            <ul>
+              <li>
+                <a href="https://twitter.com/divine_maina"
+                  ><img src="./images/./twitter.png" alt="twiter"
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="https://www.linkedin.com/in/charlotte-divine-dusenge-31b19017a/"
+                  ><img src="./images/./Linkedinicon.png" alt="linkedhin"
+                /></a>
+              </li>
+              <li>
+                <a href="mvector"
+                  ><img src="./images/./Vector.png" alt="Mvector"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/divinecharlotte"
+                  ><img src="./images/./github.png" alt="github"
+                /></a>
+              </li>
+              <li>
+                <a href="angelist"
+                  ><img src="./images/./angellist.png" alt="angellist"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <p><button type="button" class="about2">Get my resume</button></p>
+        </div>
+
+        <div class="languages">
+          
+          <div class="arrow">  
+            <h3 class="lan">Languages</h3>
+            <img class="election" src="./images/./arrow1.png" alt="" />
+          </div>
+          <ul class="mb">
+            <li><img src="./images/./Frame9(1).png" alt="image" /></li>
+            <li><img src="./images/./Frame10.png" alt="image" /></li>
+            <li><img src="./images/./Frame11.png" alt="image" /></li>
+          </ul>
+          <ul class="dsk">
+            <li><img src="./images/Frame9.png" alt="image" /></li>
+            <li><img src="./images/Frame1copy.png" alt="image" /></li>
+            <li><img src="./images/Frame11copy.png" alt="image" /></li>
+          </ul>
+          <div class="arrow">
+            <h3 class="lan">FRAMEWORKS</h3>
+            <p><img class="election2" src="./images/./left.png" alt="" /></p>
+          </div>
+          <p>
+            <img
+              class="separator"
+              src="./images/./SeparatorBottom.png"
+              alt=""
+            />
+          </p>
+          <div class="arrow">
+            <h3 class="lan">Skills</h3>
+            <p><img class="election2" src="./images/./left.png" alt="" /></p>
```

Comment:
Please try to wrap `Languages` and `FRAMEWORKS` and `skills` inside the `<ul>` tag and each one should be  `<li>` tag  as the previous reviewer requested you 
this is for semantic proposes and it will improve your SEO benchmark and your section will be structured
example : 


```html
<ul>
  <li>
    <h3>languages</h3>
    the rest of languages will be here 
</li>
  <li>
    <h3>frameworks</h3>
  </li>
  <li>
    <h3>skills</h3>
  </li>
</ul>
```